### DISCUSSION TITLE
Grouping by error message

### DISCUSSION THREAD

**Discussion by martinkwan** [getsentry/support#45] [Support] [answered]
I currently have an issue grouping Sentry Issues. Due to limitations I can't group by the default stack trace exception but instead I am using using error.message to group my Sentry Issues. However, there are variables in my error messages that create a new Sentry Issue, even if the message is fundamentally the same error (but with just different userIds for example).

Is there a way to group these similar but slightly different messages into the same issue?

**Comment by mitsuhiko** [comment, marked as answer]
You might be able to use fingerprinting rules for this: https://docs.sentry.io/product/data-management-settings/event-grouping/fingerprint-rules/

Example that matches on a message and then groups by transaction:

```
message:"system encountered a fatal problem: *" -> fatal-error, {{ transaction }}
```
